elisp/gilded-rose.el

```diff
@@ -0,0 +1,23 @@
+(defun make-item (name sell-in quality)
+  "Create an item with NAME, SELL-IN, and QUALITY."
+  (list :name name :sell-in sell-in :quality quality))
+
+(defun update-quality (item)
+  (let* ((quality (plist-get item :quality))
```

Comment:
the 3 lets are already a simplification/improvement of the original code. Is it possible to access the value from the struct each time? replace `quality``with `(plist-get item :quality)` everywhere?